decode/render.ts

```diff
@@ -8,27 +8,18 @@ let boxmodels = {};
 let metrics: MetricData = null;
 let lean = false;
 const METRIC_MAP = {};
-METRIC_MAP[Metric.Nodes] = { name: "Node Count", unit: ""};
+METRIC_MAP[Metric.Bytes] = { name: "Bytes", unit: "KB"};
 METRIC_MAP[Metric.LayoutBytes] = { name: "Layout Bytes", unit: "KB"};
 METRIC_MAP[Metric.InteractionBytes] = { name: "Interaction Bytes", unit: "KB"};
 METRIC_MAP[Metric.NetworkBytes] = { name: "Network Bytes", unit: "KB"};
-METRIC_MAP[Metric.DiagnosticBytes] = { name: "Diagnostic Bytes", unit: "KB"};
-METRIC_MAP[Metric.Mutations] = { name: "Mutation Count", unit: ""};
-METRIC_MAP[Metric.Interactions] = { name: "Interaction Count", unit: ""};
-METRIC_MAP[Metric.Clicks] = { name: "Click Count", unit: ""};
-METRIC_MAP[Metric.Selections] = { name: "Selection Count", unit: ""};
-METRIC_MAP[Metric.ScriptErrors] = { name: "Script Errors", unit: ""};
-METRIC_MAP[Metric.ImageErrors] = { name: "Image Errors", unit: ""};
-METRIC_MAP[Metric.DiscoverTime] = { name: "Discover Time", unit: "ms"};
-METRIC_MAP[Metric.MutationTime] = { name: "Mutation Time", unit: "ms"};
-METRIC_MAP[Metric.BoxModelTime] = { name: "Box Model Time", unit: "ms"};
-METRIC_MAP[Metric.StartTime] = { name: "Start Time", unit: "s"};
-METRIC_MAP[Metric.ActiveTime] = { name: "Active Time", unit: "ms"};
-METRIC_MAP[Metric.EndTime] = { name: "End Time", unit: "s"};
-METRIC_MAP[Metric.ViewportWidth] = { name: "Viewport Width", unit: "px"};
-METRIC_MAP[Metric.ViewportHeight] = { name: "Viewport Height", unit: "px"};
-METRIC_MAP[Metric.DocumentWidth] = { name: "Document Width", unit: "px"};
-METRIC_MAP[Metric.DocumentHeight] = { name: "Document Height", unit: "px"};
+METRIC_MAP[Metric.TargetBytes] = { name: "Target Bytes", unit: "KB"};
+METRIC_MAP[Metric.InvokeCount] = { name: "Invoke Count" };
+METRIC_MAP[Metric.LongTaskCount] = { name: "Long Tasks" };
+METRIC_MAP[Metric.Latency] = { name: "Latency", unit: "ms"};
```

Comment:
>Latency [](start = 18, length = 7)

This is very broad. The latency of what? Can you add a prefix?

Reply:
Called it TotalLatency and TotalBytes.